### ISSUE TITLE
Styles/CSS not loading at all

### ISSUE THREAD

**Issue by maddy2get** [telerik/nativescript-ui-samples-angular#37]
### Did you verify this is a real problem by searching [Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript) and the [other open issues in this repo](https://github.com/telerik/nativescript-ui-samples-angular/issues)?

Yes
### Tell us about the problem

Please, ensure your title is less than 63 characters long and starts with a capital
letter.

I have created new NS project with angular (`tns create sidenav --ng`) and added basic SideDrawer but after adding SideDrawer the css stopped working in the app.
### Which platform(s) does your issue occur on?

iOS
### Please provide the following version numbers that your issue occurs with:
- CLI: 2.3.0
- Cross-platform modules: 2.3.0
- Runtime(s):
  `"tns-ios": {
    "version": "2.3.0"
  }`
- Plugin(s):

```
"dependencies": {
    "@angular/common": "2.0.1",
    "@angular/compiler": "2.0.1",
    "@angular/core": "2.0.1",
    "@angular/forms": "2.0.1",
    "@angular/http": "2.0.1",
    "@angular/platform-browser": "2.0.1",
    "@angular/platform-browser-dynamic": "2.0.1",
    "@angular/platform-server": "2.0.1",
    "@angular/router": "3.0.1",
    "nativescript-angular": "1.0.1",
    "nativescript-telerik-ui-pro": "^1.4.1",
    "reflect-metadata": "^0.1.8",
    "tns-core-modules": "2.3.0"
  },
  "devDependencies": {
    "nativescript-dev-typescript": "^0.3.2",
    "typescript": "^2.0.3",
    "zone.js": "^0.6.21"
  }
```
### Please tell us how to recreate the issue in as much detail as possible.

Please clone this repo and run `tns run ios` to see the issue: https://github.com/maddy2get/ns-ng2-sidenav

Basically, I have created new NS project with --ng template and added SideDrawer to app component. Please let me know if I have missed anything while integrating SideDrawer. I have followed the documentation on integrating SideDrawer.
### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.

Yes, please see repo: https://github.com/maddy2get/ns-ng2-sidenav

**Comment by VladimirAmiorkov** [comment]
Hi @maddy2get , 

I tested the provided app project and all looks ok on my end, here is a screenshot of how the application looks. Note that the contents of the `tkDrawerContent` and `tkMainContent` all have the desired css applied, I also added the `color: green;` to the `.sideTitleStackLayout` class just to make sure.
![screenshot_20161001-133637](https://cloud.githubusercontent.com/assets/4989411/19013541/5029d26c-87dc-11e6-976f-4c3e96d78dc1.png)

If I have missed something please let me know.

**Comment by maddy2get** [comment]
Hey @VladimirAmiorkov thanks for your reply. It seems to be working fine on Android for me as well, I just tested it now but it does not work on iOS. Please see attached screenshots for both iOS and Android from my end.

I also tried deleting node_modules, lib, hooks, platforms directories and reinstall them but no luck on iOS. Hope this screenshots and info helps.

Thanks

<img width="487" alt="screen shot 2016-10-01 at 8 56 04 pm" src="https://cloud.githubusercontent.com/assets/1869939/19013641/553e4b82-881a-11e6-86a1-866787a575c3.png">
<img width="488" alt="screen shot 2016-10-01 at 8 55 46 pm" src="https://cloud.githubusercontent.com/assets/1869939/19013642/55455404-881a-11e6-85f8-e1d3d4b273ba.png">

**Comment by VladimirAmiorkov** [comment]
@maddy2get Thank you for the additional info.

I am currently out of reach of my Mac but will get back to you as soon as possible.

**Comment by VladimirAmiorkov** [comment]
Hi @maddy2get ,

I investigated the issue further and indeed the css Styles are not applied to the contents of the RadSideDrawer in the provided sample project. After further investigation it looks like this issue can be resolved by wrapping the angular 2 component that contains the RadSideDrawer inside a `page-router-outlet` like [the one](https://github.com/telerik/nativescript-ui-samples-angular/blob/release/sdkAngular/app/navigation/app.component.ts#L13) in our examples.

I created a [pull request](https://github.com/maddy2get/ns-ng2-sidenav/pull/1) that resolved the issue in the provided sample project, feel free to take a look at it.

**Comment by maddy2get** [comment]
Hey @VladimirAmiorkov,

Thanks for the pull request, that is working now.

Cheers

**Comment by VladimirAmiorkov** [comment]
@maddy2get ,

A quick updated on the state of this issue. We have resolved it and css should be applied even without the `<page-router-outlet></page-router-outlet>` being present. You can expect the fix for it to be available in our next official release **1.5.1**

**Comment by miroslavaivanova** [comment]
I will move this issue to the dedicated GitHub repository for NativeScript UI issues (as this one contains examples only): https://github.com/telerik/nativescript-ui-feedback.
If you have any feedback, issues or feature requests please log them in nativescript-ui-feedback repository.

Thanks for your understanding!
